kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -292,7 +342,7 @@
         loading,
         hasMore,
         loadingMore,
-        fetchMoreQuizResources,
+        fetchMoreQuizResources: searchQuery ? fetchMoreSearchResults : fetchMoreQuizResources,
```

Comment:
Unless the component is being torn down and recreated when the searchQuery updates (which I think we are trying to avoid happen), this value will not recompute.

You may want to instead make the fetchMoreQuizResources function behave conditionally on the searchQuery instead.

Also, it should be `searchQuery.value` I believe that we conditionalize on?